src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -513,7 +519,7 @@ blockToMarkdown' opts (CodeBlock attribs str) = do
      backticks = endline '`'
      tildes = endline '~'
      attrs  = if isEnabled Ext_fenced_code_attributes opts
-                 then nowrap $ " " <> attrsToMarkdown attribs
+                 then nowrap $ " " <> classOrAttrsToMarkdown attribs
```

Comment:
I wonder if I should keep the space in case of single class. Traditionally, there is not one.

Reply:
I like the space.

Reply:
Cool, looks like both are widely supported: https://babelmark.github.io/?text=%60%60%60+nix%0Afoo%0A%60%60%60